website/pages/en/resources/roles/delegating/undelegating.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Undelegating
+---
+
+Learn how to withdraw your delegated tokens through [Graph Explorer](https://thegraph.com/explorer) or [Arbiscan](https://arbiscan.io/).
+
+> To avoid this in the future, it's recommended that you select an Indexer wisely. To learn how to select and Indexer, check out the [Delegate](https://thegraph.com/explorer/delegate?chain=arbitrum-one) section in Graph Explorer.
+
+## How to Withdraw Using Graph Explorer
+
+### Step-by-Step
+
+1. Visit the [Graph Explorer](https://thegraph.com/explorer). Please make sure you're on Explorer and **NOT** Studio.
+2. Click on your profile. You can find it on the top right corner of the page.
+   - Make sure that your wallet is connected. If it's not connected, you will see the "connect" button instead.
+3. Once you're in your profile, click on the [Delegating](https://thegraph.com/explorer/delegate?chain=arbitrum-one) tab. In the Delegating tab, you can view the list of Indexers you have delegated to.
+4. Click on the Indexer from which you wish to withdraw your tokens.
+   - Make sure to note the specific Indexer, as you will need to find them again to withdraw.
+5. Select the "Undelegate" option by clicking on the three dots next to the Indexer on the right side, see image below:
+
+   ![Undelegate](/img/undelegate-button.png)
+
+6. After approximately [30 epochs](https://thegraph.com/explorer/network/epochs?chain=arbitrum-one) (30 days), return to the [Delegate section](https://thegraph.com/explorer/delegate?chain=arbitrum-one) and locate the specific Indexer you undelegated from.
+7. Once you find the Indexer, you can click on the Indexer and proceed to withdraw all your tokens.
+
+## How to Withdraw Using Arbiscan
+
+> This process is primarily useful if the UI in Graph Explorer experiences issues.
+
+### Step-by-Step
+
+1. Find your delegation transaction on Arbiscan.
+   - Here's an example on Arbiscan: [Transaction on Arbiscan](https://arbiscan.io/tx/0xcf2110eac897099f821064445041031efb32786392bdbe7544a4cb7a6b2e4f9a)
+2. Navigate to "Transaction Action" where you can find the staking extension contract:
+   - For the example listed above it looks like this: [Sample Contract](https://arbiscan.io/address/0x00669A4CF01450B64E8A2A20E9b1FCB71E61eF03)
+3. Then click on "Contract". ![contract on arbiscan](/img/Contract-Arbiscan.png)
```

Comment:
```suggestion
3. Then click on the "Contract" tab. ![Contract tab on Arbiscan, between NFT Transfers and Events](/img/arbiscan-contract.png)
```
(that's assuming you rename the file to `arbiscan-contract.png` like I suggested below)